### DISCUSSION TITLE
RTMP Configuration

### DISCUSSION THREAD

**Discussion by sergiobaiao** [blakeblackshear/frigate#15580] [Ask A Question]
Hi there,

I'm trying to setup the RTMP module in my HA/Frigate setup, but everytime i add this 

rtmp:
  listen: ":1935"

to Frigate configuration it won't let me save

![image](https://github.com/user-attachments/assets/1f858687-5fc9-4659-b7b0-ebdf0b31cf1d)

**Comment by NickM-27** [comment]
That's not a proper config. If you want to use rtmp you can use go2rtc that is built in to frigate.

**Reply by sergiobaiao** [reply]
is it enabled by default? Which url should i use on my camera?

i tried rtmp://192.168.0.2/live but it didn't work

**Reply by NickM-27** [reply]
See the go2rtc docs https://github.com/AlexxIT/go2rtc?tab=readme-ov-file#module-rtmp

And the frigate docs for configuring go2rtc https://docs.frigate.video/guides/configuring_go2rtc

**Reply by sergiobaiao** [reply]
that's exactly what i'm trying to achieve but i'm failing, or misunderstanding. I'm configuring my camera to send it's rtmp feed directly to Frigate, so i put on it's config:

rtmp://192.168.0.2/live (192.168.0.2 is my frigate ip address)

it won't connect. 

Is this the correct way? Or do i still need Nginx with rtmp module for this to work?

**Reply by NickM-27** [reply]
if you are trying to read an RTMP stream from a camera then you are going about it the wrong way, and you should be configuring go2rtc to connect to the camera via RTMP https://github.com/AlexxIT/go2rtc?tab=readme-ov-file#source-rtmp

**Reply by sergiobaiao** [reply]
i'm trying to send the rtmp stream from the camera to Frigate...they are on different networks, Frigate is on a public accessibly IP address while the camera is on a private network, so Frigate cannot access it directly but the camera can send the rtmp stream to an RTMP Server.
